### DISCUSSION TITLE
Blur ImGui window

### DISCUSSION THREAD

**Discussion by TrueFe3r** [ocornut/imgui#6356] [New Users Build/Link/Run issues ONLY!]
Wassup everyone, how can i blur my imgui window on opengl3 android?

void DrawMenu(){
	if(openMenuuuu){
		
		
        ImGui::SetNextWindowSize(ImVec2(menux, menuy), ImGuiCond_Once);
		
		if(!ImGui::Begin(OBFUSCATE("by @root_kalina"), NULL, ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoCollapse | ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoScrollbar)){
			ImGui::End();
			return;
		}
		
	    	ImVec2 P1, P2;
			ImDrawList* pDrawList;
			const auto& pCurrentWindowPos = ImGui::GetWindowPos();
			const auto& pWindowDrawList = ImGui::GetWindowDrawList();
			const auto& pBackgroundDrawList = ImGui::GetBackgroundDrawList();
			const auto& pForegroundDrawList = ImGui::GetForegroundDrawList();
			
			P1 = ImVec2(-100.000f, 68.000f);
			P1.x += pCurrentWindowPos.x;
			P1.y += pCurrentWindowPos.y;
			P2 = ImVec2(900.000f, 70.000f);
			P2.x += pCurrentWindowPos.x;
			P2.y += pCurrentWindowPos.y;
			pDrawList = pWindowDrawList;
			pDrawList->AddRectFilled(P1, P2, ImColor(255,255,255,255), 20);
			
			
			ImVec2 Q1, Q2;
			ImDrawList* qDrawList;
			const auto& qCurrentWindowPos = ImGui::GetWindowPos();
			const auto& qWindowDrawList = ImGui::GetWindowDrawList();
			const auto& qBackgroundDrawList = ImGui::GetBackgroundDrawList();
			const auto& qForegroundDrawList = ImGui::GetForegroundDrawList();
			
			Q1 = ImVec2(148.000f, -20.000f);
			Q1.x += qCurrentWindowPos.x;
			Q1.y += qCurrentWindowPos.y;
			Q2 = ImVec2(150.000f, 500.000f);
			Q2.x += qCurrentWindowPos.x;
			Q2.y += qCurrentWindowPos.y;
			qDrawList = qWindowDrawList;
			qDrawList->AddRectFilled(Q1, Q2, ImColor(255,255,255,255), 20);
			
			
			ImGui::SetCursorPos(ImVec2(32.0f, 12.0f));
			ImGui::Text(OBFUSCATE("Elev4te"));
			
			ImGui::SetCursorPos(ImVec2(39.0f, 28.0f));
			ImGui::Text(OBFUSCATE("0.23.1"));
			
			ImGui::SetCursorPos(ImVec2(10.0f, 80.0f));
			if(ImGui::Button(OBFUSCATE("Main"), ImVec2(128.0f, 40.0f))){
				menuss = true;
				menusss = false;
				menussss = false;
				menusssss = false;
			}
			
			ImGui::SetCursorPos(ImVec2(10.0f, 130.0f));
			if(ImGui::Button(OBFUSCATE("Visual"), ImVec2(128.0f, 40.0f))){
				menuss = false;
				menusss = true;
				menussss = false;
				menusssss = false;
			}
			
			ImGui::SetCursorPos(ImVec2(10.0f, 180.0f));
			if(ImGui::Button(OBFUSCATE("Skins"), ImVec2(128.0f, 40.0f))){
				menuss = false;
				menusss = false;
				menussss = true;
				menusssss = false;
			}
			
			ImGui::SetCursorPos(ImVec2(10.0f, 230.0f));
			if(ImGui::Button(OBFUSCATE("Misc"), ImVec2(128.0f, 40.0f))){
				menuss = false;
				menusss = false;
				menussss = false;
				menusssss = true;
			}
			
			
			
			
			if(menusssss){
				
				ImGui::SetCursorPos(ImVec2(160.0f, 80.0f));
				ImGui::Text("Menu X");
				ImGui::SetCursorPos(ImVec2(160.0f, 105.0f));
				ImGui::SliderInt("", &menux, 400, 1000);
				
				ImGui::SetCursorPos(ImVec2(160.0f, 150.0f));
				ImGui::Text("Menu Y");
				ImGui::SetCursorPos(ImVec2(160.0f, 175.0f));
				ImGui::SliderInt("", &menuy, 300, 600);
				
			    ImGui::SetCursorPos(ImVec2(160.0f, 210.0f));
				if(ImGui::Button(OBFUSCATE("Apply"), ImVec2(128.0f, 40.0f))){
	    			winSize.x = menux;
					winSize.y = menuy;
		    	}
			}
	}
	ImGui::End();
}

idk if that even possible but anyway thank you

**Comment by ocornut** [comment]
It’s been a while! Sorry but #1586
